docs/source/tensor_attributes.rst

```diff
@@ -34,13 +36,16 @@ Boolean                    ``torch.bool``                                ``torch
 To find out if a :class:`torch.dtype` is a floating point data type, the property :attr:`is_floating_point`
 can be used, which returns ``True`` if the data type is a floating point data type.
 
+To find out if a :class:`torch.dtype` is a complex data type, the property :attr:`is_complex`
```

Comment:
is this what you meant?  It returns `True` if the data type is floating point -- doesn't it return true if the data type is complex?

Reply:
no my bad. changed it to complex